tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -4961,6 +4961,159 @@ TEST_P(OpConverter_FP32_Test, ConvertSqueeze) {
+     (trt_mode_ == TrtTestMode::kImplicitBatch)
+         ? errors::Unimplemented(
+               "TensorRT does not allow modifications to "
+               "the batch dimension")
+         : Status::OK();
+   std::vector<TestParams> params = {
+     TestParams{
+           /*input_dims=*/{1, 6, 2},
+           /*begin=*/{0, 0, 0},
+           /*end=*/{0, 0, 3},
+           /*strides=*/{1, 1, 1},
+           /*begin_mask=*/get_mask({ 0,1,1}),
+           /*end_mask=*/get_mask({0,1,1}),
+           /*ellipsis_mask=*/0,
+           /*new_axis_mask=*/0,
+           /*shrink_axis_mask=*/1,
+           /*expected_output_dims=*/{6,2},
+           /*expected_output=*/{1,2,3,4,5,6,7,8,9,10,11,12},
+          /*conversion_status=*/modified_batch_dim_status,
+           Status::OK(),
+           /*partial_input_dims=*/{-1, -1, -1},
+       },
+        TestParams{
+           /*input_dims=*/{2, 3, 2},
+           /*begin=*/{0, 0, 0},
+           /*end=*/{0, 0, 3},
+           /*strides=*/{1, 1, 1},
+           /*begin_mask=*/get_mask({ 0,1,1}),
+           /*end_mask=*/get_mask({0,1,1}),
+           /*ellipsis_mask=*/0,
+           /*new_axis_mask=*/0,
+           /*shrink_axis_mask=*/1,
+           /*expected_output_dims=*/{3,2},
+           /*expected_output=*/{1,2,3,4,5,6},
+          /*conversion_status=*/modified_batch_dim_status,
+           Status::OK(),
+           /*partial_input_dims=*/{-1, -1, 2},
+       },
+       TestParams{
+             /*input_dims=*/{2, 3, 2},
+             /*begin=*/{0, 0, 0},
+             /*end=*/{0, 0, 3},
+             /*strides=*/{1, 1, 1},
+             /*begin_mask=*/get_mask({ 0,1,1}),
+             /*end_mask=*/get_mask({0,1,1}),
+             /*ellipsis_mask=*/0,
+             /*new_axis_mask=*/0,
+             /*shrink_axis_mask=*/3,
+             /*expected_output_dims=*/{2},
+             /*expected_output=*/{1,2},
+            /*conversion_status=*/modified_batch_dim_status,
+             Status::OK(),
+             /*partial_input_dims=*/{-1, -1, 2},
+         },
+ };
+
+ int i = 0;
+ for (auto p : params) {
+   Reset();
+   NodeDef node_def = get_strided_slice_nodedef(
+       tf_type_, p.begin_mask, p.end_mask, p.ellipsis_mask, p.new_axis_mask,
+       p.shrink_axis_mask);
+
+   VLOG(2) << "Preparing test case " << i++ << " with dims "
+           << DebugString(p.input_dims);
+
+   switch (trt_mode_) {
+     case TrtTestMode::kImplicitBatch: {
+       AddTestTensor("input", p.input_dims, ok_input);
+       break;
+     }
+     case TrtTestMode::kExplicitBatch: {
+       AddTestTensor("input", p.input_dims, ok_input);
+       break;
+     }
+     case TrtTestMode::kDynamicShape: {
+       if (p.partial_input_dims.size() > 0) {
+         AddTestTensor("input", p.input_dims, tf_type_, ok_input,
+                       p.partial_input_dims);
+
+       } else {
+         AddTestTensor("input", p.input_dims, tf_type_, ok_input,
+                       p.input_dims);
+       }
+       break;
+     }
+   }
```

Comment:
Could be more concise
```suggestion
       if (if trt_mode == TrtTestMode::kDynamicShape &&  p.partial_input_dims.size() > 0) {
         AddTestTensor("input", p.input_dims, tf_type_, ok_input,
                       p.partial_input_dims);
       } else {
         AddTestTensor("input", p.input_dims, ok_input); 
       }
```